### ISSUE TITLE
ON HOLD Add 533 |n to Blacklight for Sierra objects

### ISSUE THREAD

**Issue by sshetenhelm** [yalelibrary/YUL-DC#2643]
After #2640 

**Story**
In Blacklight, objects with Sierra as a source will be pulling information from 533 |n mapped to callNumberWithPublicNote in Metadata Cloud.

We would like this information to display in Blacklight for only objects with Sierra as a source. 
(Nothing needs to happen to make it only show up for Sierra on the blacklight side, since it won't be indexed for other sources.)

We would like the field added to the Blacklight parent object page and 'More Information' pane of the Universal Viewer.

The label in Blacklight should be called Call Number with Public Note _(can we easily change this later if needed?)_

**Acceptance**
- [ ] Add callNumberWithPublicNote_tsim to the search_fields list in Blacklight for call number search
- [ ] Add Call Number with Public Note to display in Blacklight on parent object page, directly below 'Call Number'
- [ ] Add Call Number with Public Note to display in Universal Viewer 'More Information' pane, directly below 'Call Number'

**Comment by jpengst** [comment]
NOTE: I dont think we have a Sierra feature branch for Blacklight yet, so we will need to make one.

**Comment by sshetenhelm** [comment]
Law has requested we put a hold on this at this point in time. They have requested re-mapping 533 |n to Call Number, as originally worked. As such, no need to add this field in Blacklight.

Will save this ticket, as Law mentioned potentially revisiting this issue in the future.
